torchvision/prototype/models/googlenet.py

```diff
@@ -14,14 +14,13 @@
 
 
 class GoogLeNetWeights(Weights):
-    ImageNet1K_Community = WeightEntry(
+    ImageNet1K_TFV1 = WeightEntry(
         url="https://download.pytorch.org/models/googlenet-1378be20.pth",
         transforms=partial(ImageNetEval, crop_size=224),
         meta={
             "size": (224, 224),
             "categories": _IMAGENET_CATEGORIES,
             "interpolation": InterpolationMode.BILINEAR,
-            "recipe": "https://github.com/TheCodez/examples/blob/inception/imagenet/README.md#googlenet",
```

Comment:
I Think in other PRs we did `"recipe": ""`, should we align this in the end as well?

Reply:
Good spot. So there are no longer `"recipe": ""` because in an earlier commit I actually added all of them. Only one of them doesn't support it, so I've set it to None:
https://github.com/pytorch/vision/blob/6a60b9bccc4faf48b9738f0f3906d1b8108fd03a/torchvision/prototype/models/densenet.py#L70

But your comment made me realize that for a few TF models that don't have a recipe, I haven't set the key to `None`. Should I omit if not available or set it to `None`? Let me know and I'll align the behaviour everywhere in this PR.